sdk/action.go

```diff
@@ -365,6 +361,34 @@ func UpdateJoinedAction(projectKey, pipelineName string, stage int, a Action) er
 	return nil
 }
 
+//LoadAction creates an action on CDS
```

Comment:
Yeah, you're right. This name is same than handler's name on api, and this handler creates an action after "loading" a file (url or file). Now, this handler can insert or update an existing action, with a json data POST by sdk, this is simpliest than post a multipart form, not common in sdk. So name is not great anymore. You may have a new name proposition ? upsertAction ?